products/jse-dept/features/jse_dept_related_content.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Ensure Site Users can view, select, and reset the filter by related content section on the news and events page
+  In order to ensure that any site user can view, select, and reset the filter by related content section on the news and events page
+  As a Site User
+  I want to be able to view, select, and reset the filter by related content section on the news and events page
+
+  Background:
+     Given I am on "News & Events"
```

Comment:
The news and events pages were split into separate pages so I'll create different tests for each